packages/www/src/web-components/play-ground.tsx

```diff
@@ -1,17 +1,18 @@
-import type { WebContext } from 'brisa';
+import "@spectrum-web-components/split-view/sp-split-view.js";
```

Comment:
This works and is fine. However, so that the web component can be consumed anywhere (every server component, etc) and you don't need to add the `packages/www/src/global.d.ts` type declaration, you can do it this way:

https://brisa.build/building-your-application/components-details/external-libraries#third-party-brisa-web-components-vs-native-web-components

The beauty of this implementation is that it only adds the import when it is consumed and its integration is done throughout the app. It also allows to add better types (attributes, etc), and for the WC of breeze its part of SSR.

It is only a comment, it is not necessary that you change it for this specific case it is well, I comment it simply so that you know that exist this way.

Reply:
Exactly I anticipated that, therefore asked about this. I am excited for this framework ❤️ Thank you very much for the explanation.